lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -57,10 +59,10 @@ def serializable_hash_for_collection(serializer, options)
           hash
         end
 
-        def serializable_hash_for_single_resource(serializer, options)
+        def serializable_hash_for_single_resource(options)
           primary_data = primary_data_for(serializer, options)
           relationships = relationships_for(serializer)
-          included = included_for(serializer)
+          included = included_resources(@include_tree)
```

Comment:
my concern with ivars is that they're more buggy. If I type `@includ_tree` I get some method called on nil error which I have to track down.  If I call `includ_tree`, it blows up at load.  So, I tend not to make them outside of the initializers or getters and setters.  I get that there's probably some performance hit